cli/aiconfig-editor/src/shared/prompt_schemas/OpenAIChatModelParserPromptSchema.ts

```diff
@@ -66,6 +66,7 @@ export const OpenAIChatModelParserPromptSchema: PromptSchema = {
         type: "integer",
         maximum: 4096,
         minimum: 16,
+        default: 4096,
```

Comment:
Another benefit of using the callback instead of `useEffect` is that we can now safely set defaults in the schema without them immediately being propagated to the config